extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as os from 'os';
+import { isNullOrUndefined } from 'util';
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+export const DBProjectConfigurationKey: string = 'sqlDatabaseProjects';
+export const NetCoreInstallLocationKey: string = 'netCoreSDKLocation';
+export const NetCoreInstallationConfirmation: string = localize('sqlDatabaseProjects.NetCoreInstallationConfirmation', "The .NET Core SDK cannnot be located. Project build will not work. Please install the same or update the .Net Core SDK location in settings if already installed.");
```

Comment:
Do we (ADS) have a preference or a best practice for whether UI strings should be extracted to a constants/loc file or kept in the same file?  I've mostly seen them get extracted

Reply:
Checked in code codebase not one practice as such - since these are very few and used here only - I prefer to keep is close to code.